build.sbt

```diff
@@ -0,0 +1,70 @@
+import NativePackagerHelper._
+
+ThisBuild / turbo := true
+
+enablePlugins(JavaAppPackaging)
+enablePlugins(DebianPlugin)
+
+lazy val viper_api = (project in file("viper"))
+lazy val parsers = (project in file("parsers"))
+
+lazy val vercors = (project in file("."))
+    .dependsOn(viper_api)
+    .dependsOn(parsers)
+    .settings(
+        name := "Vercors",
+        organization := "University of Twente",
+        version := "1.1.0",

```

Comment:
Do we need to change this to 1.2.0?

Reply:
Good catch, yes.

Reply:
I guess this small change does not deserve a full travis test. So I will merge now.